docs/reference/overview.md

```diff
@@ -31,6 +38,12 @@ Setting this is optional. If you do not set this, the `COMPOSE_PROJECT_NAME` def
 
 Specify the file containing the compose configuration. If not provided, Compose looks for a file named  `docker-compose.yml` in the current directory and then each parent directory in succession until a file by that name is found.
 
+### COMPOSE\_API\_VERSION
+
+Specify the api version the client should report. If you get an error about
+`client and server don't have same version` you may use this environment
+variable to have the client report a version compatible with the server.
```

Comment:
Hmm, the intent has changed a bit with this wording and now it's not really correct, how about:

> The Docker API only supports requests from clients which report a specific version.  If you receive a `client and server don't have same version` error using `docker-compose`, setting this environment variable prevents this error. Set it to the version you want the client to report. typically the server version.

> **NOTE**: Setting a different API version will prevent some features from working, and is not officially supported.